unix-ffi/unittest/unittest.py

```diff
@@ -444,7 +444,7 @@ def _test_cases(mod):
         c = getattr(mod, tn)
         if isinstance(c, object) and isinstance(c, type) and issubclass(c, TestCase):
             yield c
-        elif tn.startswith("test_") and callable(c):
+        elif tn.startswith("test") and callable(c):
```

Comment:
Sorry for breaking some tests with this change... I don't remember clearly now but I think I found some existing test scripts in the repo where there were testFoo functions being run that were not intended to be unittest functions, so thought the `_` made more sense. I believe this is non-standard functionality that cpython doesn't support anyway (finding & running bare test* functions)?

Reply:
> I believe this is non-standard functionality that cpython doesn't support anyway (finding & running bare test* functions)?

I think so yes, at least I didn't find anything which does that. I'd personally refrain from doing that because it's non-standard, and because if you have a file like that then decide it runs under CPython as well you might think its tests are all fine but they are just not being ran. CPython does support a single `runTest` function though like

```
class Test(unittest.TestCase):
  @staticmethod
  def runTest():
    test = unittest.TestCase()
    test.assertEqual(1, 1)
```

But anyway `test` is the standard prefix; e.g. [`By default these are the method names beginning with test`](https://docs.python.org/3/library/unittest.html#loading-and-running-tests) and `testMethodPrefix = 'test'` in `unittest/loader.py`.